docs/content/faq.md

```diff
@@ -87,3 +87,27 @@ Which currently includes:
 - hg
 - bzr
 - fossil
+
+### When should I use a vendor directory, and when should I use Athens?
+
+The Go community has used vendor directories for a long time before module proxies like Athens came along, so naturally each group collaborating on code should decide for themselves whether they want to use a vendor directory, use Athens, or do both!
+
+Vendor (without a proxy) are valuable when:
+
+- CI/CD systems don't have access to an Athens (even if it's internal)
+- When the vendor directory is so small that it is still faster to check it out from a repo than it is to pull zip files from the server
+- If you're coming from glide/dep or another dependency management system that leveraged the vendor directory
+
+Athens valuable when:
```

Comment:
Can you write this as `Athens (without a vendor directory) is valuable when:`? I want the docs to be clear that we're talking about just a proxy, and deleting the entire vendor directory